packages/@lwc/engine-server/src/__tests__/fixtures/scoped-slots/default-slot/modules/x/parent/parent.html

```diff
@@ -0,0 +1,9 @@
+<template>
+    <x-child>
+        <!-- TODO [#5020]: Fix rendering of scoped slot content, so that content outside of the template renders correctly with engine-server -->
```

Comment:
This comment is great! I have a feeling there are other tests added recently which are expected failures of engine-server but aren't documented as such and only in PR comments.